These changes add three new error numbers for the sake of RFS.
***************
*** 90,95
  #define	EUSERS		68		/* Too many users */
  #define	EDQUOT		69		/* Disc quota exceeded */
  
  
  /*-----------------System V unique errno's----------------------*/
  #define	ENOMSG	100	/* No message of desired type		*/

--- 90,99 -----
  #define	EUSERS		68		/* Too many users */
  #define	EDQUOT		69		/* Disc quota exceeded */
  
+ /* remote fs stuff */
+ #define EISREMOTE	70		/* file is on a remote system */
+ #define ETOOMANYREMOTE	71		/* too many remote file systems */
+ #define	ENOREMOTEFS	72		/* no remote file system */
  
  /*-----------------System V unique errno's----------------------*/
  #define	ENOMSG	100	/* No message of desired type		*/
